src/fields/address.coffee

```diff
@@ -0,0 +1,24 @@
+FormRenderer.Models.ResponseFieldAddress = FormRenderer.Models.ResponseField.extend
```

Comment:
These per-field files were pulled from response_fields.coffee and main.coffee into their own files, with some extra functionality added to ResponseField (e.g. validators) to be shared across all of them. This was Adam's original refactor, and was entangled + low-impact enough that I figured it would be safer to leave it in than to extract what was relevant to repeating sections and rework the sections related to including these fields in them.